lumigator/frontend/src/components/organisms/LDatasetDetails.vue

```diff
@@ -47,36 +51,36 @@
       </div>
       <div class="l-dataset-details__content-item">
         <span class="l-dataset-details__content-label">submitted</span>
-        <span class="l-dataset-details__content-field">
+        <span class="l-dataset-details__content-field" v-if="selectedDataset">
```

Comment:
the whole component should probably not be rendered if there's no `selectedDataset` as it can just receive that as a required prop instead of having these checks everywhere, but that's an opportunity for future refactoring